### DISCUSSION TITLE
img2img masking causing artifacts?

### DISCUSSION THREAD

**Discussion by Hunterius8** [hlky/stable-diffusion-webui#378] [General]
Is it just me, or does the img2img masking now cause some pretty severe artifacting in areas that should remain untouched based on the mask settings? I placed a mask around the weird triangle set to regenerate only the masked area, first image is the base image, then the next one after a single img2img mask run, and the third one is what I ended up with after generating and then picking one of the generated images 5 times.

![base](https://user-images.githubusercontent.com/25130543/187719299-cd4007ab-dcaa-4be3-85f9-ae512ccef9aa.png)
![1x](https://user-images.githubusercontent.com/25130543/187719379-5fdf17b6-87c9-4179-896d-84784b552995.png)
![5x](https://user-images.githubusercontent.com/25130543/187719414-9447055a-85e0-4036-b11b-16ba93be62a0.png)

**Comment by anon-hlhl** [comment]
Some further discussion around this here: https://github.com/hlky/stable-diffusion/issues/153

**Comment by mchaker** [comment]
There is a fix in the `img2img-fix` branch. See comment: https://github.com/hlky/stable-diffusion-webui/issues/664#issuecomment-1237025304
